serving/src/main/resources/application.yml

```diff
@@ -1,12 +1,51 @@
 feast:
-  # This value is retrieved from project.version properties in pom.xml
-  # https://docs.spring.io/spring-boot/docs/current/reference/html/
-  version: @project.version@
   # GRPC service address for Feast Core
   # Feast Serving requires connection to Feast Core to retrieve and reload Feast metadata (e.g. FeatureSpecs, Store information)
   core-host: ${FEAST_CORE_HOST:localhost}
   core-grpc-port: ${FEAST_CORE_GRPC_PORT:6565}
 
+  # Indicates the active store. Only a single store in the last can be active at one time. In the future this key
+  # will be deprecated in order to allow multiple stores to be served from a single serving instance
+  active_store: online
+
+  # List of store configurations
+  stores:
```

Comment:
As there are several changes in configurations, should this still be ported into v0.4 branch? On the other hand, if we don't port this, any PR that follows after this PR is merged will have a much higher chance of conflicting with the v0.4 branch.

Reply:
There are loads of breaking changes. If we backport this then we will have a lot of problems. 

> any PR that follows after this PR is merged will have a much higher chance of conflicting with the v0.4 branch

That is true, but that is why we maintain a separate branch for 0.4. We should ideally move on from 0.4 because the amount of changes is high and its not worth the upkeep.

Reply:
I agree that we should move on from 0.4. But in that case, do we still want to release 0.4.8? And if we do, what should be the last commit that we should stop at? Storage API refactoring?

Reply:
The point is that we dont backport anything significant. Only critical functionality like hotfixes. Otherwise there is no point in having releases if we just backport everything.